RELEASE_NOTES.md

```diff
@@ -475,6 +475,10 @@ See <https://medium.com/rubyinside/whats-new-in-ruby-2-7-79c98b265502> for detai
 
 Ohai has been improved to gather additional system configuration information for use when authoring recipes and resources.
 
+#### filesystem2 Node Data available on Windows
+
+In previous Chef Infra Clients we've introduced a modernized filesystem layout of Ohai data for many platforms. In Chef 16.0, Windows now has this layout available in `node['filesystem2']`. In Chef 17, it will replace `node['filesystem']` to match all other platforms.
```

Comment:
```suggestion
In previous Chef Infra Clients we've introduced a modernized filesystem layout of Ohai data for many platforms. In Chef Infra Client 16.0, Windows now has this layout available in `node['filesystem2']`. In Chef Infra Client 17, it will replace `node['filesystem']` to match all other platforms.
```